[2013-09-25 12:59:51] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'date_default_timezone_set' (T_STRING), expecting function (T_FUNCTION)' in D:\vhosts\laraveil\app\library\daty.php:12
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:00:45] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dateMod()' in D:\vhosts\laraveil\app\library\daty.php:97
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:02:06] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dateMod()' in D:\vhosts\laraveil\app\library\daty.php:99
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:08:46] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dateMod()' in D:\vhosts\laraveil\app\library\daty.php:102
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:09:11] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dateMod()' in D:\vhosts\laraveil\app\library\daty.php:102
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:09:14] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dateMod()' in D:\vhosts\laraveil\app\library\daty.php:102
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:09:23] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'l' (T_STRING), expecting ',' or ';'' in D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8:93
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:11:38] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dateMod()' in D:\vhosts\laraveil\app\library\daty.php:115
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:12:39] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function dateMod()' in D:\vhosts\laraveil\app\library\daty.php:115
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:17:32] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '->' (T_OBJECT_OPERATOR)' in D:\vhosts\laraveil\app\library\daty.php:109
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:18:11] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Using $this when not in object context' in D:\vhosts\laraveil\app\library\daty.php:115
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:22:04] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Using $this when not in object context' in D:\vhosts\laraveil\app\library\daty.php:115
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:22:16] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Using $this when not in object context' in D:\vhosts\laraveil\app\library\daty.php:115
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:22:47] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Using $this when not in object context' in D:\vhosts\laraveil\app\library\daty.php:115
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:31:51] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::showTimeAgo() should not be called statically, assuming $this from incompatible context' in D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c:15
Stack trace:
#0 D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c(15): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 15, Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#6 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#13 {main} [] []
[2013-09-25 13:32:39] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::get_page_mod_time() should not be called statically, assuming $this from incompatible context' in D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8:93
Stack trace:
#0 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(93): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 93, Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c(21): Illuminate\View\View->render()
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#11 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#12 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#13 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#14 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2013-09-25 13:32:55] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::get_page_mod_time() should not be called statically, assuming $this from incompatible context' in D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8:93
Stack trace:
#0 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(93): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 93, Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c(21): Illuminate\View\View->render()
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#11 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#12 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#13 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#14 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2013-09-25 13:33:00] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Using $this when not in object context' in D:\vhosts\laraveil\app\library\daty.php:115
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 13:36:14] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::showTimeAgo() should not be called statically, assuming $this from incompatible context' in D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c:15
Stack trace:
#0 D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c(15): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 15, Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#6 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#13 {main} [] []
[2013-09-25 13:40:08] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::showTimeAgo() should not be called statically, assuming $this from incompatible context' in D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c:15
Stack trace:
#0 D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c(15): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 15, Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#6 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#13 {main} [] []
[2013-09-25 13:40:15] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::showTimeAgo() should not be called statically, assuming $this from incompatible context' in D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c:15
Stack trace:
#0 D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c(15): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 15, Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#6 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#13 {main} [] []
[2013-09-25 13:43:48] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Daty::dateMod()' in D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8:93
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:28:27] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'date_default_timezone_set' (T_STRING), expecting function (T_FUNCTION)' in D:\vhosts\laraveil\app\library\dateMod.php:11
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:28:53] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'echo' (T_ECHO), expecting function (T_FUNCTION)' in D:\vhosts\laraveil\app\library\dateMod.php:49
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:29:04] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected end of file, expecting function (T_FUNCTION)' in D:\vhosts\laraveil\app\library\dateMod.php:47
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:38:28] log.ERROR: exception 'ErrorException' with message 'Object of class Carbon\Carbon could not be converted to int' in D:\vhosts\laraveil\app\library\daty.php:35
Stack trace:
#0 D:\vhosts\laraveil\app\library\daty.php(35): Illuminate\Exception\Handler->handleError(8, 'Object of class...', 'D:\vhosts\larav...', 35, Array)
#1 D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c(15): Daty::showTimeAgo(Object(Carbon\Carbon))
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#7 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2013-09-25 14:39:59] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function getMinutes()' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:46:21] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function getMinutes()' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:47:27] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function getMinutes()' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:47:28] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function getMinutes()' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:49:12] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:31)' in D:\vhosts\laraveil\app\library\daty.php:31
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:49:31] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:14)' in D:\vhosts\laraveil\app\library\daty.php:14
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:49:33] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:14)' in D:\vhosts\laraveil\app\library\daty.php:14
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:49:34] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:14)' in D:\vhosts\laraveil\app\library\daty.php:14
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:49:49] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:14)' in D:\vhosts\laraveil\app\library\daty.php:14
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:49:50] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:14)' in D:\vhosts\laraveil\app\library\daty.php:14
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:49:50] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:14)' in D:\vhosts\laraveil\app\library\daty.php:14
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:49:50] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:14)' in D:\vhosts\laraveil\app\library\daty.php:14
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:49:51] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:14)' in D:\vhosts\laraveil\app\library\daty.php:14
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:51:00] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare getMinutes() (previously declared in D:\vhosts\laraveil\app\library\daty.php:14)' in D:\vhosts\laraveil\app\library\daty.php:14
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:52:15] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '->' (T_OBJECT_OPERATOR)' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:52:28] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Using $this when not in object context' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 14:52:50] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Using $this when not in object context' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:00:11] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::showTimeAgo() should not be called statically, assuming $this from incompatible context' in D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c:15
Stack trace:
#0 D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c(15): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 15, Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#6 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#13 {main} [] []
[2013-09-25 15:08:49] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'getMinutes' not found' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:14:44] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Daty::get_page_mod_time()' in D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8:93
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:15:17] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined method Daty::get_page_mod_time()' in D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8:93
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:15:22] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ')', expecting ',' or ';'' in D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8:93
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:15:55] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function getMinutes()' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:23:42] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function getMinutes()' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:23:55] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Carbon' not found' in D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8:93
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:24:38] log.ERROR: exception 'InvalidArgumentException' with message 'The separation symbol could not be found
The separation symbol could not be found
A two digit minute could not be found
Unexpected data found.
Trailing data' in D:\vhosts\laraveil\vendor\nesbot\carbon\src\Carbon\Carbon.php:333
Stack trace:
#0 D:\vhosts\laraveil\vendor\nesbot\carbon\src\Carbon\Carbon.php(276): Carbon\Carbon::createFromFormat('Y-n-j G:i:s', '1380115478-9-25...', NULL)
#1 D:\vhosts\laraveil\vendor\nesbot\carbon\src\Carbon\Carbon.php(291): Carbon\Carbon::create(1380115478, NULL, NULL, NULL, NULL, NULL, NULL)
#2 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(93): Carbon\Carbon::createFromDate(1380115478)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#7 D:\vhosts\laraveil\app\storage\views\148d7498c71ea05e85766c9b4396673c(21): Illuminate\View\View->render()
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#12 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#13 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#14 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#15 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#16 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#17 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#19 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#20 {main} [] []
[2013-09-25 15:28:42] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function getMinutes()' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:33:59] log.ERROR: exception 'ErrorException' with message 'Undefined variable: self' in D:\vhosts\laraveil\app\library\daty.php:46
Stack trace:
#0 D:\vhosts\laraveil\app\library\daty.php(46): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', 'D:\vhosts\larav...', 46, Array)
#1 D:\vhosts\laraveil\app\storage\views\7b1eea9ce07082b54f9b9d0720e9bc3f(42): Daty::showTimeAgo(Object(Carbon\Carbon))
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#7 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2013-09-25 15:35:39] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::showTimeAgo() should not be called statically, assuming $this from incompatible context' in D:\vhosts\laraveil\app\storage\views\7b1eea9ce07082b54f9b9d0720e9bc3f:4
Stack trace:
#0 D:\vhosts\laraveil\app\storage\views\7b1eea9ce07082b54f9b9d0720e9bc3f(4): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 4, Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#6 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#13 {main} [] []
[2013-09-25 15:36:50] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::showTimeAgo() should not be called statically, assuming $this from incompatible context' in D:\vhosts\laraveil\app\storage\views\7b1eea9ce07082b54f9b9d0720e9bc3f:4
Stack trace:
#0 D:\vhosts\laraveil\app\storage\views\7b1eea9ce07082b54f9b9d0720e9bc3f(4): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 4, Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#6 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#13 {main} [] []
[2013-09-25 15:36:56] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function getMinutes()' in D:\vhosts\laraveil\app\library\daty.php:76
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2013-09-25 15:37:32] log.ERROR: exception 'ErrorException' with message 'Non-static method Daty::getMinutes() should not be called statically' in D:\vhosts\laraveil\app\library\daty.php:76
Stack trace:
#0 D:\vhosts\laraveil\app\library\daty.php(76): Illuminate\Exception\Handler->handleError(2048, 'Non-static meth...', 'D:\vhosts\larav...', 76, Array)
#1 D:\vhosts\laraveil\app\library\daty.php(76): Daty::getMinutes()
#2 D:\vhosts\laraveil\app\storage\views\7b1eea9ce07082b54f9b9d0720e9bc3f(42): Daty::showTimeAgo(Object(Carbon\Carbon))
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#8 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#12 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#13 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#14 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#15 {main} [] []
[2013-09-25 16:04:58] log.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::order_by()' in D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php:1635
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('order_by', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->order_by('created_at', 'desc')
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(755): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('order_by', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->order_by('created_at', 'desc')
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Relations\Relation.php(271): call_user_func_array(Array, Array)
#6 D:\vhosts\laraveil\app\models\Article.php(12): Illuminate\Database\Eloquent\Relations\Relation->__call('order_by', Array)
#7 D:\vhosts\laraveil\app\models\Article.php(12): Illuminate\Database\Eloquent\Relations\HasMany->order_by('created_at', 'desc')
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(507): App\Models\Article->comments()
#9 [internal function]: Illuminate\Database\Eloquent\Builder->Illuminate\Database\Eloquent\{closure}()
#10 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Relations\Relation.php(159): call_user_func(Object(Closure))
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(508): Illuminate\Database\Eloquent\Relations\Relation::noConstraints(Object(Closure))
#12 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(476): Illuminate\Database\Eloquent\Builder->getRelation('comments')
#13 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(456): Illuminate\Database\Eloquent\Builder->loadRelation(Array, 'comments', Object(Closure))
#14 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(120): Illuminate\Database\Eloquent\Builder->eagerLoadRelations(Array)
#15 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(89): Illuminate\Database\Eloquent\Builder->get(Array)
#16 D:\vhosts\laraveil\app\routes.php(51): Illuminate\Database\Eloquent\Builder->first()
#17 [internal function]: {closure}('pierwszy-post')
#18 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#19 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#20 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#21 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#22 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#23 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#24 {main} [] []
[2013-09-25 18:10:21] log.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1044] Access denied for user 'czarnodz_1'@'94.251.220.202' to database 'czarnodz_laraveil'' in D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:47
Stack trace:
#0 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(47): PDO->__construct('mysql:host=185....', 'czarnodz_1', 'vSyJqZ1r', Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=185....', Array, Array)
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(41): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(115): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(64): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(2374): Illuminate\Database\DatabaseManager->connection(NULL)
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(2340): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1443): Illuminate\Database\Eloquent\Model->getConnection()
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1377): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 D:\vhosts\laraveil\vendor\cartalyst\sentry\src\Cartalyst\Sentry\Users\Eloquent\Provider.php(75): Illuminate\Database\Eloquent\Model->newQuery()
#10 D:\vhosts\laraveil\vendor\cartalyst\sentry\src\Cartalyst\Sentry\Sentry.php(266): Cartalyst\Sentry\Users\Eloquent\Provider->findById(1)
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(174): Cartalyst\Sentry\Sentry->check()
#12 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(24): Illuminate\Support\Facades\Facade::__callStatic('check', Array)
#13 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(24): Cartalyst\Sentry\Facades\Laravel\Sentry::check()
#14 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#15 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#16 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#17 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#18 D:\vhosts\laraveil\app\storage\views\656796fb4e0169ca571889b7569ab4d8(21): Illuminate\View\View->render()
#19 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#20 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#21 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#22 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#23 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#24 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#25 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#26 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#27 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#28 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#29 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#30 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#31 {main} [] []
[2013-09-25 18:10:57] log.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1044] Access denied for user 'czarnodz_1'@'94.251.220.202' to database 'czarnodz_laraveil'' in D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:47
Stack trace:
#0 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(47): PDO->__construct('mysql:host=185....', 'czarnodz_1', 'vSyJqZ1r', Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=185....', Array, Array)
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(41): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(115): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(64): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(2374): Illuminate\Database\DatabaseManager->connection(NULL)
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(2340): Illuminate\Database\Eloquent\Model::resolveConnection(NULL)
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1443): Illuminate\Database\Eloquent\Model->getConnection()
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1377): Illuminate\Database\Eloquent\Model->newBaseQueryBuilder()
#9 D:\vhosts\laraveil\vendor\cartalyst\sentry\src\Cartalyst\Sentry\Users\Eloquent\Provider.php(75): Illuminate\Database\Eloquent\Model->newQuery()
#10 D:\vhosts\laraveil\vendor\cartalyst\sentry\src\Cartalyst\Sentry\Sentry.php(266): Cartalyst\Sentry\Users\Eloquent\Provider->findById(1)
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(174): Cartalyst\Sentry\Sentry->check()
#12 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(24): Illuminate\Support\Facades\Facade::__callStatic('check', Array)
#13 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(24): Cartalyst\Sentry\Facades\Laravel\Sentry::check()
#14 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#15 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#16 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#17 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#18 D:\vhosts\laraveil\app\storage\views\656796fb4e0169ca571889b7569ab4d8(21): Illuminate\View\View->render()
#19 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#20 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#21 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#22 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#23 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#24 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#25 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#26 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#27 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#28 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#29 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#30 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#31 {main} [] []
[2013-09-25 18:12:07] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'czarnodz_laraveil.users' doesn't exist' in D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connection.php:271
Stack trace:
#0 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connection.php(271): PDO->prepare('select * from `...')
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connection.php(520): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'select * from `...', Array)
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connection.php(276): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1007): Illuminate\Database\Connection->select('select * from `...', Array)
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(997): Illuminate\Database\Query\Builder->runSelect()
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(984): Illuminate\Database\Query\Builder->getFresh(Array)
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(422): Illuminate\Database\Query\Builder->get(Array)
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(113): Illuminate\Database\Eloquent\Builder->getModels(Array)
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(89): Illuminate\Database\Eloquent\Builder->get(Array)
#9 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(64): Illuminate\Database\Eloquent\Builder->first(Array)
#10 D:\vhosts\laraveil\vendor\cartalyst\sentry\src\Cartalyst\Sentry\Users\Eloquent\Provider.php(75): Illuminate\Database\Eloquent\Builder->find(1)
#11 D:\vhosts\laraveil\vendor\cartalyst\sentry\src\Cartalyst\Sentry\Sentry.php(266): Cartalyst\Sentry\Users\Eloquent\Provider->findById(1)
#12 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(174): Cartalyst\Sentry\Sentry->check()
#13 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(24): Illuminate\Support\Facades\Facade::__callStatic('check', Array)
#14 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(24): Cartalyst\Sentry\Facades\Laravel\Sentry::check()
#15 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#16 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#17 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#18 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#19 D:\vhosts\laraveil\app\storage\views\656796fb4e0169ca571889b7569ab4d8(21): Illuminate\View\View->render()
#20 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#21 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#22 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#23 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#24 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#25 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#26 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#27 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#28 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#29 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#30 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#31 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#32 {main}

Next exception 'Exception' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'czarnodz_laraveil.users' doesn't exist (SQL: select * from `users` where `id` = ? limit 1) (Bindings: array (
  0 => 1,
))' in D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connection.php:555
Stack trace:
#0 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connection.php(528): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'select * from `...', Array)
#1 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Connection.php(276): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#2 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1007): Illuminate\Database\Connection->select('select * from `...', Array)
#3 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(997): Illuminate\Database\Query\Builder->runSelect()
#4 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(984): Illuminate\Database\Query\Builder->getFresh(Array)
#5 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(422): Illuminate\Database\Query\Builder->get(Array)
#6 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(113): Illuminate\Database\Eloquent\Builder->getModels(Array)
#7 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(89): Illuminate\Database\Eloquent\Builder->get(Array)
#8 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(64): Illuminate\Database\Eloquent\Builder->first(Array)
#9 D:\vhosts\laraveil\vendor\cartalyst\sentry\src\Cartalyst\Sentry\Users\Eloquent\Provider.php(75): Illuminate\Database\Eloquent\Builder->find(1)
#10 D:\vhosts\laraveil\vendor\cartalyst\sentry\src\Cartalyst\Sentry\Sentry.php(266): Cartalyst\Sentry\Users\Eloquent\Provider->findById(1)
#11 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(174): Cartalyst\Sentry\Sentry->check()
#12 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(24): Illuminate\Support\Facades\Facade::__callStatic('check', Array)
#13 D:\vhosts\laraveil\app\storage\views\0dbff89585fefafd55b59e5c8b2096f8(24): Cartalyst\Sentry\Facades\Laravel\Sentry::check()
#14 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#15 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#16 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#17 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#18 D:\vhosts\laraveil\app\storage\views\656796fb4e0169ca571889b7569ab4d8(21): Illuminate\View\View->render()
#19 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\vhosts\larav...')
#20 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\vhosts\larav...', Array)
#21 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\vhosts\larav...', Array)
#22 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#23 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Http\Response.php(69): Illuminate\View\View->render()
#24 D:\vhosts\laraveil\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#25 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#26 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#27 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#28 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#29 D:\vhosts\laraveil\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#30 D:\vhosts\laraveil\public\index.php(49): Illuminate\Foundation\Application->run()
#31 {main} [] []
